scripts/create-package/plopfile.ts

```diff
@@ -86,7 +86,7 @@ module.exports = (plop: NodePlopAPI) => {
       if (answers.target === 'react') answers = { hasTests: true, ...answers };
       const { packageName, target, hasTests } = answers;
 
-      const destination = `packages/${packageName}`;
+      const destination = answers.isConverged ? `packages/react-components/${packageName}` : `packages/${packageName}`;
```

Comment:
**q:** this scaffolding is valid only for v9 right? so in future we can get rid of that isConverged flag ?

Reply:
Yup and going forward it's most likely going to be used to create v9 components anyway